contest/views.py

```diff
@@ -113,11 +115,11 @@ def contest(request, cid):
 
 @login_required
 def new(request):
+    title = "New Contest"
     if can_create_contest(request.user):
         if request.method == 'GET':
             form = ContestForm(initial=\
-                {'user':request.user, 'method':request.method})
-            title = "New Contest"
+                {'owner':request.user, 'user':request.user, 'method':request.method})
```

Comment:
owner is used for default value when creating contest
user is for problem list shown while creating contest
I think this will be more unified